app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+  def create
```

Comment:
You did great job! I like the implementation you did!

- [ ] However, I spot an error in your comments controller. The error is you comments should have a `new` method like post controller has.

Reply:
it is created just below it, otherwise, you won't even be able to write a new comment!! 😐 
<img width="335" alt="issue1" src="https://user-images.githubusercontent.com/77294268/235138797-780e3485-0f35-4e9e-8281-9f443d60fd56.png">